sdk/storage/AzureStorageBlob/Source/DataStructures/StorageStreamDownloader.swift

```diff
@@ -24,6 +24,10 @@
 //
 // --------------------------------------------------------------------------
 
+// swiftlint:disable file_length
+// swiftlint:disable function_body_length
```

Comment:
I would prefer to keep these in issue #198 as I feel they are legitimate and we could break these into smaller chunks. Ditto for elsewhere the file-level disable applies.

Reply:
Agree, I would have preferred to actually fix the issues in this PR. However it gets tricky because a lot of the function_length violations are when we set up a call, then make the request, and then have a callback for the completion. I went through a couple options but they would have made semantic changes rather than just cosmetic ones, and I didn't want to introduce semantic changes in this PR.

Reply:
I agree, that's why I'm just saying don't mark them complete in the tracking issue :)

Reply:
Ahh okay, I understand what you mean. I definitely want to keep all of the ones that have been disabled in-code in the tracking issue until we determine that we feel strongly about keeping a given disable rather than making the fix.